src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -15,6 +17,22 @@ public class SearchDictionary<V>
         private ILogger logger;
         private static int LIMIT_SEARCH_TAG_SIZE = 300;
 
+        static SearchDictionary()
+        {
+            try
+            {
+                // Look up search tag limit in the assembly configuration
+                var assemblyConfig = ConfigurationManager.OpenExeConfiguration(Assembly.GetExecutingAssembly().Location);
```

Comment:
Is this the `DynamoCore.dll.config` file?

Reply:
yes